pom.xml

```diff
@@ -69,7 +70,8 @@
                             apps/pantheon;overwrite:=true;uninstall:=true;path:=/apps/pantheon,
                             SLING-INF/content;overwrite:=false;uninstall:=false;path:=/content,
                             SLING-INF/users;overwrite:=false;uninstall:=false,
```

Comment:
You are right. I wasn't ware we already had a users file.
I'll need to figure out how to create a system user and modify the users.json file. :+1:

Reply:
So, per the sling documentation, this mechanism doesn't allow for the creation of service users :disappointed: . One of the recommended ways is to do it in the `SlingRepositoryInitializer` class. I propose we tackle this in a separate PR as it might need some thinking and for now have the restriction that a service account is a pre-requisite?

Reply:
Ah, shucks! Ok, I like your suggestion to create the user in the `SlingRepositoryInitializer` class. That's a good compromise.

Reply:
Oh! I see you already have a new commit that does exactly this. Nice!